### ISSUE TITLE
Broken Links in documentation

### ISSUE THREAD

**Issue by anselal** [Chalarangelo/mini.css#100]
Hey there,

let me first say that I really love your project. I have downloaded your latest release and was ready to read the documentation but the links on the navbar do not link to the respective html files. I think that the links are made so that they work only for your minicss.org domain but not for offline reading.
Would it be ok if a create a pull request and change the links so that they will work offline ?

**Comment by thornjad** [comment]
It looks like they are linking to the files, but they're not using the `.html` extension. In a browser, that looks better and is what I would have done. However, it doesn't work if you're just opening the HTML files from your computer.

I'm not @Chalarangelo, but my suggestion would be, instead of a PR, to run a server on localhost. My preferred way to do this is with the npm package `http-server`. If you have Node installed, use `npm install --global http-server`, and then run `http-server` in the `docs` directory of mini.css. By default, the site will then be up at `http://localhost:8080`, and will handle the links correctly.

**Comment by Chalarangelo** [comment]
Hi, @anselal. Thank you very much for the support and for bringing this to my attention! Truth be told, I rerouted all the links after I got the custom domain for the project to make it look better. However, when working on the offline documentation, this has been an issue for me in the past, so I did what @Raindeer44 suggests - using a lightweight server locally. I mean you are going to be building your project anyways, so you might as well use a server to test it before deploying it.

My tool of preference is [Web Server for Chrome](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb?hl=en) as it is really lightweight and easy to use (although I am not a Chrome user). Alternatively, I would suggest just browsing the docs online, as they are going to be always up-to-date (sometimes the docs are changed even if there is not a new release).

A PR would be slightly out of place, especially now that **Gluon** is in development and lots of the documentation is being moved around the branches (I will be deploying the **v3** documentation in parallel for people to browse and give me feedback).

In short, this will be remedied over time, as **Gluon** rolls out, because I will make it a PWA so that you won't really have to get the docs. However, I might set some time aside to write a small script (or somebody else might offer to undertake the project) to create an offline version of the docs, as **Fermion** will not be a PWA and some people might still want it.

The **current plan** is to find some time next time I update the docs (in a few days) to create a zip for people like you who want the documentation for offline use and add it to the latest release, so that this will not be an issue. Does that sound like a good plan to you?

**Comment by anselal** [comment]
Firing up a webserver just to browse simple html files **is not a** pretty **solution**. I will rather open the html files one by one on my browser.

**Comment by Chalarangelo** [comment]
@anselal truth be told, it's not too pretty. That's why I will bundle the documentation in a zip with proper links, just to be on the safe side! ;)

**Comment by anselal** [comment]
Thank you for the support and congratulations for the :100:+ reported issues :tada:

**Comment by anselal** [comment]
- [ ] At http://minicss.org/flavors the link for the Google's Material design color palette needs the .html extension to work

**Comment by Chalarangelo** [comment]
@anselal It worked at the time I wrote the page, but thanks a lot for spotting this. I will eventually get around to writing a crawler/fetcher to check all the links, but it has always seemed like too much work.

Fixing this as soon as I've had my morning coffee! 😅

**Comment by anselal** [comment]
Good. I was looking into the different flavors and just stepped on it. A crawler is a good idea. You can do it just with regular expressions or python's BeautifulSoap library if you are familiar.

**Comment by Chalarangelo** [comment]
Link is fixed! 

I was thinking about using Python cause it sounds fun, I just never got around to doing it because I am usually preoccupied with other projects, but I think now would be a good time, especially with **Gluon** on the way making the documentation a terrifying mess to maintain.

**Comment by anselal** [comment]
Quick as always !!! Yeah, python is fun and easy to use.
If you need any help just ask ;)
